.gitignore

```diff
@@ -1,3 +1,6 @@
 /vendor/
 /composer.lock
 /bin/
+
+*.sublime-project
```

Comment:
These should be in local, global `.gitignore`.

Reply:
These were added by someone else, I don't use Sublime :)

Reply:
My bad. Sorry.